Guided Hunting - Azure Resource Explorer.ipynb

```diff
@@ -0,0 +1,7311 @@
+{
```

Comment:
<p>Maybe delete the output from this cell before checkin</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>done!</p>